waspc/src/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -38,30 +39,32 @@ genLogout = C.copyTmplAsIs (C.asTmplFile [relfile|src/auth/logout.js|])
 
 -- | Generates HOC that handles auth for the given page.
 genCreateAuthRequiredPage :: Wasp.Auth.Auth -> FileDraft
-genCreateAuthRequiredPage auth =
-  C.makeTemplateFD
-    (asTmplFile $ [reldir|src|] </> authReqPagePath)
-    targetPath
-    (Just templateData)
-  where
-    authReqPagePath = [relfile|auth/pages/createAuthRequiredPage.js|]
-    targetPath = C.webAppSrcDirInWebAppRootDir </> (asWebAppSrcFile authReqPagePath)
-    templateData = object ["onAuthFailedRedirectTo" .= (Wasp.Auth._onAuthFailedRedirectTo auth)]
+genCreateAuthRequiredPage auth = [relfile|auth/pages/createAuthRequiredPage.js|] `compiledWithData` ["onAuthFailedRedirectTo" .= (Wasp.Auth._onAuthFailedRedirectTo auth)]
```

Comment:
Also let's break it into multiple lines. Btw we are using ormolu for formatting, so I would recommend also using that!

Reply:
Yes, our style guide is to have max 100 cols in a line. Although it would also be nice if we somewhere published it publicly, that's maybe one thing to do :)

Reply:
Ideally we would enforce that automatically, via some tool, but I am not sure how to best do that at the moment!